data/Music/Monkey Island Series/The Secret Of Monkey Island/Theme.meta

```diff
@@ -0,0 +1,2 @@
+The Secret of Monkey Island - Theme
+0x03
```

Comment:
I don't actually know what this number means. Can someone help me out with that?

Reply:
Is this maybe to distinguish fanfares from background music?

Reply:
The number distinguishes which instrumentalist the game is supposed to use.

Reply:
Ah okay, so this is main orchestra. How does this value interact with the instruments in the midi file?

Reply:
it says the rando what instrument list to load.

and depending on the list chosen the values in the sequence file have to be changed cause there are different instruments obviously

Reply:
Thanks for the help!
-- I switched up the instruments for the Secret of Monkey Island theme: https://www.youtube.com/watch?v=JH6VoP8QM7A
-- I added a Fanfare from Monkey Island 2 (Chapter Screen): https://www.youtube.com/watch?v=vDaztQ67MQk
-- I also switched up the Monkey Island 2 Theme

Unfortunately, the Monkey Island 2 Theme is now too large to be played in most places, so I plan to trim that down at some point in the future.